src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String ballColor, int ballNumber) {
+        this.color = ballColor;
+        this.number = ballNumber;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{"
+                + "color=" + color
+                + ", number" + number + '\''
```

Comment:
It seems there is a typo in the toString method. There should be an '=' sign after 'number' to correctly output the number field value.